AirTote/Services/ImagerySourceReader.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+namespace AirTote.Services;
+
+public class ImagerySourceReader
+{
+	public string GroupName { get; set; }
+	public string ShortTitle { get; set; }
+	public string FullTitle { get; set; }
+	public string URL { get; set; }
```

Comment:
バグ防止のため、dotnetからの警告に従って初期値を設定してください。

```suggestion
	public string GroupName { get; set; } = "";
	public string ShortTitle { get; set; } = "";
	public string FullTitle { get; set; } = "";
	public string URL { get; set; } = "";
```